ONSdigital/census-rm-qid-batch-runner
# fix descriptions for D_CCS_CHP1 and D_CCS_CHP2W products

# Motivation and Context
The descriptions in the manifest for the D_CCS_CHP1 and D_CCS_CHP2W products are wrong

# What has changed
change the descriptions for D_CCS_CHP1 and D_CCS_CHP1

# How to test?
run a batch for D_CCS_CHP1 and D_CCS_CHP1 products, using this config:
`Pack code,Questionnaire type,Quantity
D_CCS_CHP1,51,75
D_CCS_CHP2W,53,80`

you should then see the correct descriptions in the generated manifest files
